### ISSUE TITLE
Books Page: mobile Redesign & cleanup

### ISSUE THREAD

**Issue by jimchamp** [internetarchive/openlibrary#6484]
This is an epic for tracking outstanding book page issues from the canonical book page desktop redesign #684 as well as the remaining mobile redesign steps. 

@hayoonc helped lead a mobile redesign of the Books Page in figma: https://www.figma.com/file/uIY20MIY2LgwKT7KwCgYJB/Open-Library-Mobile?node-id=0%3A1

We've mostly converged on this v1 design (with some proposed changes to desktop to support a mobile-first approach):
![www figma com_file_uIY20MIY2LgwKT7KwCgYJB_Open-Library-Mobile_node-id=0%3A1 (6)](https://user-images.githubusercontent.com/978325/191997758-fdb650d7-4873-4851-a1db-b835b74b7f1e.png)

## Next Steps

Several next steps are outlined in the **decisions** section of our planning document here:
https://docs.google.com/document/d/1buJASy3MbxQPYstoUGW_tAwC7UYTKfcejUm8xXsfAIc/edit#heading=h.2c6wwj2hb9ei

Also, we may want to combine these related issues 
- [ ] #3561
- [x] #3607

## Subtasks

- [x] @jimchamp to update this list based on the **decisions** [here](https://docs.google.com/document/d/1buJASy3MbxQPYstoUGW_tAwC7UYTKfcejUm8xXsfAIc/edit#heading=h.2c6wwj2hb9ei)
- [x] #6538
- [x] #6537
- [ ] #6536
- [x] #6535
- [x] #6534
- [x] #7024
- [x] #7417

This epic will track the following Book Page Hacktoberfest issues:

- [ ] #7004
- [ ] #7005
- [x] #7006`
- [x] #7007
- [x] #7008
- [x] #7009

**Comment by mekarpeles** [comment]
# Books Page: Update Header Section Design 

Please refer here to our "v1" figma designs:
https://www.figma.com/file/uIY20MIY2LgwKT7KwCgYJB/Open-Library-Mobile?node-id=940-5115&t=tn4iWAkrchGgt5u3-0

### Describe the problem that you'd like solved


* [ ] Add share button to mobile header
* [ ] Removes the grey underlined work-level title
* [ ] Makes the book title font-size from  2em -> ~1.5em
* [ ] Cleans star ratings & moves extras below the fold
* [ ] Remove underline on author name + decrease font size
* [ ] On mobile, remove grey border around book cover -- `.Tools .btn-notice, .read-options`
* [ ] Convert `Preview` button into preview link

## Before
<img style="flex:1" width="403" alt="Screenshot 2023-03-20 at 2 54 30 PM" src="https://user-images.githubusercontent.com/978325/226474109-ea5508e4-df42-41cd-a74b-55f1e098b108.png">

<img width="293" alt="Screenshot 2023-03-20 at 3 07 02 PM" src="https://user-images.githubusercontent.com/978325/226476134-26008ba6-99d3-487f-9aba-68cc16e48035.png">

## After
<img style="flex:1" width="566" alt="Screenshot 2023-03-20 at 2 51 34 PM" src="https://user-images.githubusercontent.com/978325/226473624-31145a6e-7fe9-465c-9b65-10059944a522.png">

<img width="471" alt="Screenshot 2023-03-20 at 2 52 07 PM" src="https://user-images.githubusercontent.com/978325/226473708-b2de8797-4e66-43ba-8346-032c2ee99739.png">

<img width="541" alt="Screenshot 2023-03-20 at 2 52 32 PM" src="https://user-images.githubusercontent.com/978325/226473765-a2eb6d11-a494-40a5-9ab4-a742130594de.png">

### Additional context


* The code for the book page template is located here:
https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/type/edition/view.html

* The stylesheets are located here:
https://github.com/internetarchive/openlibrary/tree/master/static/css

* Here's a video on developing the Books Page:
https://archive.org/details/openlibrary-tour-2020/book-page-developers-guide.mp4

* Here's our front-end developer's guide
https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide

**Comment by kmcgonigle** [comment]
@mekarpeles & @jimchamp I'll be able to work on the remaining styling To Do's outlined above this week. Please let me know if any of these changes are already in flight, or if it's safe to open a PR for all of them. Thank you!  🙂

**Comment by jimchamp** [comment]
@kmcgonigle nothing in [this comment](https://github.com/internetarchive/openlibrary/issues/6484#issuecomment-1477009991) is in flight, as far as I know.

This work is being done on the `book-page-integration` [branch](https://github.com/internetarchive/openlibrary/tree/book-page-integration).  Please pull from there before starting.

**Comment by cdrini** [comment]
@mekarpeles @jimchamp Would be good to determine what of this checklist above are blockers to getting this integration branch merged into master. Anything that can be done in a separate issue it would be nice to pullout into a new issue so we can get that merged in by end of sprint.

**Comment by kmcgonigle** [comment]
@cdrini I'm not sure when your current sprint ends, but I was working on these updates over the weekend and will be opening a draft PR for review shortly -- I'll tag you in it as well.

**Comment by kmcgonigle** [comment]
@jimchamp I just opened a draft PR for feedback. I did run into a few specific questions while working on certain pieces:

- Removes the grey underlined work-level title
  - Should this be moved somewhere, or just hidden? (Right now, just hidden with `display:none`)
- Makes the book title font-size from 2em -> ~1.5em
  - Should subtitle text (when it exists) also become smaller? Sized this text down to `1em` for now.
- Cleans star ratings & moves extras below the fold
  -  Is the ratings text (next to star rating) a clickable link (text in comp is blue — I have not made this color change yet)
- Remove underline on author name + decrease font size
  - I removed the underline (and decreased font size) for mobile only; should the `text-decoration` be consistent on both desktop and mobile?
- Convert Preview button into preview link
  - This isn’t as important on mobile, but should there be a hover state for this link?

I also noticed that on the comps, the width of the Borrow, Want to Read, etc. buttons seems to be constrained (instead of full-width as it is now). Not sure if that update was needed as well -- just let me know.
